shared/census_mapping.json

```diff
@@ -2,27 +2,27 @@
   "age": {
     "aggregated_ages.zero_to_eighteen_percent": {
       "joined_column_name": "zero_to_eighteen",
-      "human_readable_name": "0-18 Years"
```

Comment:
We use a space around - for cash "$15k - $50k", maybe we should use it for ages as well?